src/test/shell/integration/configured_query_test.sh

```diff
@@ -1416,4 +1416,73 @@ EOF
   expect_not_log "QueryException"
 }
 
+function test_files_output_mode() {
```

Comment:
We're trying to move tests into Java where possible.

Would the patterns in https://github.com/bazelbuild/bazel/blob/c3f20f00d43a4c3811137cb4740623e254e45b1e/src/test/java/com/google/devtools/build/lib/query2/cquery/BuildOutputFormatterCallbackTest.java#L120-L129 and related files be able to capture this coverage?

Reply:
Definitely, will redo the test and cover all the features. I just didn't want to write extensive tests for something that may not get merged, but now that this is viewed favorably, I'm going to test it thoroughly.